scripts/linters/pylint_extensions.py

```diff
@@ -264,47 +263,74 @@ class HangingIndentChecker(checkers.BaseChecker):
         ),
     }
 
-    def process_module(self, node):
-        """Process a module.
+    def process_tokens(self, tokens):
+        """Process tokens to check if there is a line break after the bracket.
 
         Args:
-            node: astroid.scoped_nodes.Function. Node to access module content.
+            tokens: astroid.Tokens. Object to process tokens.
         """
-        file_content = read_from_node(node)
-        file_length = len(file_content)
-        exclude = False
-        for line_num in python_utils.RANGE(file_length):
-            line = file_content[line_num].lstrip().rstrip()
-            # The source files are read as bytes, hence the b' prefix.
-            if line.startswith(b'"""') and not line.endswith(b'"""'):
-                exclude = True
-            if line.endswith(b'"""'):
-                exclude = False
-            if line.startswith(b'#') or exclude:
-                continue
-            line_length = len(line)
-            bracket_count = 0
-            for char_num in python_utils.RANGE(line_length):
-                char = line[char_num]
-                if char == b'(':
-                    if bracket_count == 0:
-                        position = char_num
-                    bracket_count += 1
-                elif char == b')' and bracket_count > 0:
-                    bracket_count -= 1
-            if bracket_count > 0 and position + 1 < line_length:
-                content = line[position + 1:]
-                if not len(content) or not b',' in content:
-                    continue
-                split_list = content.split(b', ')
-                if len(split_list) == 1 and not any(
-                        char.isalpha() for char in split_list[0]):
-                    continue
-                separators = set('@^! #%$&)(+*-=')
-                if not any(char in separators for item in split_list
-                           for char in item):
+        escape_character_indicator = b'\\'
+        string_indicator = b'\''
+        for (token_type, token, (line_num, _), _, line) in tokens:
+            # Check if token type is an operator and is either a
+            # left parenthesis '(' or a right parenthesis ')'.
+            if token_type == tokenize.OP and (
+                    token == b'(' or token == b')'):
+                line = line.strip()
+                bracket_count = 0
+                line_length = len(line)
+                escape_character_found = False
+                in_string = False
+                for char_num in python_utils.RANGE(line_length):
+                    char = line[char_num]
+                    # Check if we are inside a string and if a escape
+                    # character is present then do not check the escape
+                    # character and the character that follows next because
+                    # next character may be a string indicator (') and
+                    # that flip the boolean in_string to False and but that
```

Comment:
I don't think we need an explanation of why we are not flipping the in_string here. Also, if you think it is good to add it here then please change this line to: 'The boolean in_string is not set to false because we are still inside a string'. Example...

It is really upto you to keep this comment :).